recipes/mupdf/Makerules.patch

```diff
@@ -0,0 +1,11 @@
+--- Makerules.orig	2020-03-26 15:39:59.725834877 -0500
++++ Makerules	2020-03-26 15:40:10.141808906 -0500
+@@ -62,7 +62,7 @@
+ endif
+ 
+ # Default system libraries
+-SYS_FREETYPE_LIBS := -lfreetype2
++SYS_FREETYPE_LIBS := -lfreetype
```

Comment:
You can do `make SYS_FREETYPE_LIBS="-lfreetype"` and remove this patch

Reply:
That would involve having separate make lines for linux and mac and I thought a small patch was cleaner